### PR TITLE
Add mss as dependency and use for screenshotting

### PR DESCRIPTION
**Pull request by xylix**
BREAKING CHANGES: removes ´save_format´ argument from take_screenshot and `crop_image` functions / Robot keywords

Using mss improves screenshot delay / latency by almost 90% (on my machine changes from 2-2.5 seconds to 150-250ms).

I created a new test, which reveals (at least on my machine) that find by template finds the screenshot at a different location from the one specified when the screenshot was taken. This is a problem with matching templates, but the speed is heavily improved. I'll try to run the tests on windows as well to see if that works nicely.

TODO:
- [x] Remove now unnecessary pyscreenshot dependency
- [x] test on Windows

Manually tested that `Take Screenshot` and `Crop Image` work on windows and macOS. Also added a new test, but it has some issues with an offset in "find by template" (the coordinate space for the find_by function might be flipped or something, but that's out of scope for this PR)

### CODE REVIEW

File: packages/main/src/RPA/Images.py [hunk 1]
```diff
@@ -163,35 +163,38 @@ def __init__(self):
...
+
         region = to_region(region)
 
-        if region is not None:
-            image = ImageGrab.grab(bbox=region.as_tuple())
-        else:
-            image = ImageGrab.grab()
+        with mss.mss() as sct:
+            if region is not None:
+                image = sct.grab(region.as_tuple())
```
Comment (osrjv) [inline packages/main/src/RPA/Images.py:177]: Is the origin the top-left of the main monitor or the whole virtual screen?

File: packages/main/src/RPA/Images.py [hunk 2]
```diff
@@ -200,7 +203,8 @@ def crop_image(self, image, region, filename=None, save_format="PNG"):
         image.load()
 
         if filename:
-            image.save(filename, save_format)
+            # Suffix isn't created automatically here
+            image.save(filename + ".png", "PNG")
```
Comment (osrjv) [inline packages/main/src/RPA/Images.py:207]: should this also use pathlib?

File: packages/main/src/RPA/Images.py [hunk 3]
```diff
@@ -243,7 +247,7 @@ def find_template_in_image(
         if not matches:
             raise ImageNotFoundError("No matches for given template")
 
-        # Convert region coördinates back to full-size coördinates
```
Comment (osrjv) [inline packages/main/src/RPA/Images.py:246]: Oh no, you removed my favourite stupid spelling: https://en.wiktionary.org/wiki/co%C3%B6rdinate#English

**Review by osrjv — CHANGES_REQUESTED** [review]
Talking about conventions, check that `make lint` passes. We auto-format everything through black and I think some of the line removals will error out.

File: packages/main/src/RPA/Images.py (hunk 3 above)
Comment (xylix) [reply packages/main/src/RPA/Images.py:246]: Oh my :D I'll undo the change

File: packages/main/src/RPA/Images.py (hunk 2 above)
Comment (xylix) [reply packages/main/src/RPA/Images.py:207]: Yeah that's probably a more stable solution

File: packages/main/src/RPA/Images.py (hunk 1 above)
Comment (xylix) [reply packages/main/src/RPA/Images.py:177]: I am pretty sure the origin is the top-left of the whole virtual screen. mss docs were a bit lean on this so I'm unsure, will try to test

**Pull request merged**